tests/models/whisper/test_modeling_whisper.py

```diff
@@ -1269,25 +1269,6 @@ def test_generate_with_prompt_ids_and_task_and_language(self):
         for row in output.tolist():
             self.assertListEqual(row[: len(expected_output_start)], expected_output_start)
 
-    def test_generate_with_prompt_ids_and_forced_decoder_ids(self):
```

Comment:
We might still want to test that generating with `forced_decoder_ids` and `prompt_ids` give the right answer, aligned with the original Whisper.

Do we already have a test like that or should we do it ourselves ?

Reply:
We have tests that verify that the output is aligned when setting `prompt_ids=`: `test_whisper_longform_prompt_ids`, `test_generate_with_prompt_ids`. Yet, we do (and did) not have any when setting `forced_decoder_ids` (the one we had was just checking that the generated sequence had the forced ids in the beginning). Let me add one!

Reply:
test added in commit [e5064d9](https://github.com/huggingface/transformers/pull/34111/commits/e5064d9b74ce5847075d52245d8414dea427dffe)